src/Microsoft.OpenApi.OData.Reader/Common/Constants.cs

```diff
@@ -59,5 +59,10 @@ internal static class Constants
         /// extension for group type
         /// </summary>
         public static string xMsDosGroupPath = "x-ms-docs-grouped-path";
+
+        /// <summary>
+        /// extension for paging
+        /// </summary>
+        public static string xMsPageable = "x-ms-pageable";
```

Comment:
Can you add a config for this extension, so that the existing customer is not affected

Reply:
#Resolved.